kube-client/src/client/mod.rs

```diff
@@ -238,15 +237,17 @@ impl Client {
         Ok(text)
     }
 
-    /// Perform a raw HTTP request against the API and get back the response
-    /// as a stream of bytes
-    pub async fn request_text_stream(
-        &self,
-        request: Request<Vec<u8>>,
-    ) -> Result<impl Stream<Item = Result<Bytes>>> {
+    /// Perform a raw HTTP request against the API, get back the response
+    /// and return it as an AsyncBufRead. Users can call `.lines()` on it
```

Comment:
I think the docs should link to `AsyncReadExt` and `AsyncBufReadExt`, `AsyncRead` does not have a `.lines()` itself.

Reply:
```
    /// Perform a raw HTTP request against the API, get back the response
    /// and return it as an object that implements [`AsyncBufRead`] and thus
    /// [`AsyncBufReadExt`]. Users can call `.lines()` on it to get a newline
    /// buffered stream of logs.
```

does this work?

Reply:
That looks sensible to me. You could make it into the standard format:

```
/// Headline
///
/// Extra text only shown when browsing to the item
```

not a big deal though, you can see how it ends up looking with `just doc` locally